src/uu/env/Cargo.toml

```diff
@@ -18,6 +18,9 @@ path = "src/env.rs"
 clap = { workspace = true }
 rust-ini = { workspace = true }
 uucore = { workspace = true, features = ["signals"] }
+subst = { path="../../../submodules/subst" }
```

Comment:
why submodules ?

Reply:
This is meant to be temporary... I initially tried to use a git submodule.
Anyway, it will be removed as soon as we have a decition on what we can change on subst side.
Discussion with the subst-author is ongoing: https://github.com/fizyr/subst/pulls